src/classes/VOL_CTRL_VolunteersJobListingFS_TEST.cls

```diff
@@ -30,8 +30,113 @@
 
 @isTest
 private with sharing class VOL_CTRL_VolunteersJobListingFS_TEST {
+    private static VOL_CTRL_VolunteersJobListingFS jobListingCtrl;
+    private static final Integer queryLimit = VOL_CTRL_VolunteersJobListingFS.SUBQUERY_LIMIT;
```

Comment:
Should this constant also be upper case?

Reply:
@scottwarren-sfdo I asked myself the same question! I recently read that if the constant is a hard coded value then it should be all upper case but if it will not know it's value until run time then it should be regular camelCase but if you don't agree I'd be ok with all upper case.

Reply:
Spoke with @jjbennett and we did some research. To keep in alignment with other teams, we're going to capitalize constants regardless of calculated / referential nature.